### ISSUE TITLE
Training code release plan

### ISSUE THREAD

**Issue by gaojh0333** [Robbyant/lingbot-map#81]
Thank you for your excellent work. When will the training code be released?

**Comment by LinZhuoChen** [comment]
Sorry for reply late. For now, we don't have plans to open-source the training code. As a starting point, you could refer to the VGGT training framework and use Claude to assist in implementing the attention mask with MagiAttention.
